src/help/changelog.md

```diff
@@ -1,6 +1,6 @@
-#### March 19, 2021{section=changelog}
+#### April 5, 2021{section=changelog}
 In this release of the Broker, we:
 
-* Added a warning alerting the user to unsaved comments on the lower section of the Review & Publish page.
-* Updated the text on several FABS validations to match DAIMS more closely and updated minor bug in FABS 38.2.2 logic to function correctly. 
-* Added a note on the Release Note and Tech Note Archive pages to indicate that historical links are not maintained.
+* Added another warning notification on the Review & Publish page to alert users when they have unsaved comments.
+* Updated FREC agency names to use the FREC abbreviation instead of the parent CGAC abbreviation.
+* Fixed a bug on the Error and Warning report where an incorrect flex field value was being returned in a few instances for certain validations.
```

Comment:
Did we do that this sprint? I thought we just determined that it was no longer an issue and didn't actually change anything.

Reply:
Good point. I think that was a misunderstanding of a JIRA ticket status. We can remove it.